classes/Import.php

```
@@ -0,0 +1,160 @@
+    }
+
+    /**
+     * Stores all CSV rows in a short-lived transient and returns the transient key.
+     *
+     * @param array $rows All rows from the parsed CSV.
+     * @return string Transient key.
+     */
+    public static function storeRows(array $rows): string
+    {
+        $key = self::TRANSIENT_PREFIX . wp_generate_password(12, false);
+        set_transient($key, $rows, 30 * MINUTE_IN_SECONDS);
```

PHPStan (2.2.2)

[error] Constant MINUTE_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)